hphp/test/frameworks/results/mockery.expect

```diff
@@ -727,8 +739,6 @@ Mockery_MockTest::testShouldAllowMockingProtectedMethodReturnsMockInstance
 NamedMockTest::itCreatesANamedMock
 .
 NamedMockTest::itCreatesPassesFurtherArgumentsJustLikeMock
-FATAL
-NamedMockTest::itShouldThrowIfAttemptingToRedefineNamedMock
```

Comment:
Maybe the hhvm team have changed how recording works since this was last recorded?